### PR TITLE
PyTorch ROCm back!

### PR DESCRIPTION
**Pull request by Jorghi12**
PyTorch ROCm builds fail right now since some introduced C++ files contain HIP headers.

### CODE REVIEW

File: caffe2/CMakeLists.txt [hunk 1]
```diff
@@ -260,13 +260,19 @@ endif()
 
 # ---[ Caffe2 HIP sources.
 if(USE_ROCM)
-  # Call again since Caffe2_HIP_INCLUDES is extended with ATen include dirs.
   IF(BUILD_ATEN)
+     # Call again since Caffe2_HIP_INCLUDES is extended with ATen include dirs.
      HIP_INCLUDE_DIRECTORIES(${Caffe2_HIP_INCLUDES})
+
+     # Necessary since some of ATen's C++ files contain HIP includes.
+     SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -D__HIP_PLATFORM_HCC__=1")
```
Comment (bddppq) [inline caffe2/CMakeLists.txt]: wait is it hipify adding those includes?

Comment (Jorghi12) [reply caffe2/CMakeLists.txt]: Nope. There's just Cpp files being generated by the build system that contain CUDA headers.

**Comment by ezyang** [comment]
Is this PR still relevant? PyTorch ROCm seems to be passing on master.

**Comment by ezyang** [comment]
obsolete

**Pull request closed without merging**